transcribe.py

```diff
@@ -75,13 +76,20 @@ def transcribe_file(audio_path, tlog_path):
                 json.dump(transcripts, tlog_file, default=float)
 
 
-def transcribe_many(path_pairs):
-    pbar = create_progressbar(prefix='Transcribing files | ', max_value=len(path_pairs)).start()
-    for i, (src_path, dst_path) in enumerate(path_pairs):
-        p = Process(target=transcribe_file, args=(src_path, dst_path))
+def transcribe_many(paths, kind):
+    pbar = create_progressbar(prefix='Transcribing files | ', max_value=len(paths)).start()
+    if kind == 'dir':
+        # the user pointed to a dir of files
+        src_paths = paths
+        dst_paths = [ path.replace('.wav','.tlog') for path in paths ]
+    elif kind == 'catalog':
+        # the user pointed to a catalog dir from DSAlign
+        src_paths,dst_paths = zip(*paths)
```

Comment:
Why so complicated? This `if` block and the `kind` parameter are not required if all the case-specific preparations would just be done in the corresponding cases in `main` - just before calling `transcribe_many`.

Reply:
un-complicat-ified the function:)